tests/TenantDatabaseManagerTest.php

```diff
@@ -220,6 +220,91 @@
 
     $manager->setConnection('mysql2');
     expect($manager->databaseExists($name))->toBeTrue();
+
+    $manager->deleteDatabase($tenant);
+    expect($manager->databaseExists($name))->toBeFalse();
+});
+
+test('tenant database can be created on a foreign server by using the host from tenant config', function () {
+    config([
+        'tenancy.database.managers.mysql' => MySQLDatabaseManager::class,
+        'tenancy.database.template_tenant_connection' => 'mysql',
```

Comment:
Why are you configuring `mysql` instead of `mysql2` here?

Reply:
Is it that the tenant's `tenancy_db_host` will override the hostname here? And therefore use the `mysql2` host instead of what's specified in `template_tenant_connection`?

Reply:
> Is it that the tenant's tenancy_db_host will override the hostname here? And therefore use the mysql2 host instead of what's specified in template_tenant_connection?

Yes.

Reply:
I see. Add a comment at the end of line 231 saying `// This will be overridden by tenancy_db_host`.

And similar to the other review, check if removing the `mysql2` connection details works.

Reply:
> Add a comment at the end of line 231

Done

> And similar to the other review, check if removing the mysql2 connection details works.

Removing the `mysql2` connection config does not work.

Reply:
I see, seems like that's something we only used in `TenantDatabaseManagerTest` (in master). It's fine to configure it here then.

Reply:
Ok. Resolving the review now.